### DISCUSSION TITLE
Nx Angular with dynamic module federation doesn't respects webpack.prod.config remotes config

### DISCUSSION THREAD

**Discussion by bhanuchhabra7** [nrwl/nx#13772] [Show and Tell]
I have created this setup using nx angular dynamic module federation.

host
    |
    ---- app1
    ---- app2
    
The host has webpack.prod.config and /assets/module-federation.manifest.json

content of webpack.prod.config
``` javascript
const { withModuleFederation } = require('@nrwl/angular/module-federation');
const config = require('./module-federation.config');
module.exports = withModuleFederation({
  ...config,
  remotes:[
    ['app1', 'https://app1.example.com'],
    ['app2', 'https://app2.example.com']
  ]
```
contents of 
module-federation.manifest.json

``` json
{
  "app1": "http://localhost:4201",
  "app2": "http://localhost:4202"
}
```

main.ts looks like this:

import { setRemoteDefinitions } from '@nrwl/angular/mf';
``` javascript
fetch('/assets/module-federation.manifest.json')
  .then((res) => res.json())
  .then((definitions) => setRemoteDefinitions(definitions))
  .then(() => import('./bootstrap').catch((err) => console.error(err)));
```

then I build all apps using
``` shell
nx run app2:build:production
nx run app1:build:production
nx run host:build:production
```

then I serve application using any server (say IIS or Nginx)

I get this error:
![image](https://user-images.githubusercontent.com/16737272/207126095-7e2e611e-451c-4da2-9a26-911783c202f5.png)

**EXPECTATION** : when building the host app in production mode, the configuration provided in webpack.prod.config should be respected.

**Comment by Noxichek** [comment]
@bhanuchhabra7 Do u resolve this problem?
